css-positioning/css-positioning-quiz-notes.md

```diff
@@ -0,0 +1,46 @@
+# css-positioning-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the **default** value for the `position` property of HTML elements?
+normal flow
+- How does setting `position: relative` on an element affect document flow?
+other things stay in its intended position
+- How does setting `position: relative` on an element affect where it appears on the page?
+able to adjust it by shifting in a direction.
+- How does setting `position: absolute` on an element affect document flow?
+does not affect normal flow of other elements
```

Comment:
Not quite true. Check out how other elements move when you give something `position: absolute`